app/Services/GoCardlessBankData.php

```
@@ -0,0 +1,354 @@
+        return $this->processTokenResponse($response);
+    }
+
+    /**
+     * Parses the token response, updates access and refresh tokens, and sets their expiration times.
+     *
+     * @param  mixed  $response  The HTTP response containing token data.
+     * @return string The new access token.
+     */
+    private function processTokenResponse($response): string
+    {
+        $data = $response->json();
```

PHPStan (2.1.15)

Cannot call method json() on mixed.


(method.nonObject)